src/main/resources/templates/datewarehouse/datawarehouse-config.json

```diff
@@ -1277,7 +1188,188 @@
+			"scriptName": "team_snapshots.py",
+			"sourcePath" : "log.sagebase.org/teamSnapshots/records/",
+			"targetTable": "teamsnapshots"
+		},
+		{
+			"name": "teammembersnapshots",
+			"description": "This job processes the team member snapshot data",
+			"scriptName": "team_member_snapshots.py",
+			"sourcePath" : "log.sagebase.org/teamMemberSnapshots/records/",
+			"targetTable": "teammembersnapshots"
+		},
+		{
+			"name": "usergroupsnapshots",
+			"description": "This job processes the user group snapshot data",
+			"scriptName": "user_group_snapshots.py",
+			"sourcePath" : "log.sagebase.org/userGroupSnapshots/records/",
+			"targetTable": "usergroupsnapshots"
+		},
+		{
+			"name": "userprofilesnapshots",
+			"description": "This job processes the user profile snapshot data",
+			"scriptName": "user_profile_snapshots.py",
+			"sourcePath" : "log.sagebase.org/userProfileSnapshots/records/",
+			"targetTable": "userprofilesnapshots"
+		},
+		{
+			"name": "fileuploadrecords",
+			"description": "This job processes the file upload records",
+			"scriptName": "file_upload_records.py",
+			"sourcePath" : "log.sagebase.org/fileUploadRecords/records/",
+			"targetTable": "fileuploadrecords"
+		},
+		{
+			"name": "filedownloadrecords",
+			"description": "This job processes the file download records",
+			"scriptName": "file_download_records.py",
+			"sourcePath" : "log.sagebase.org/fileDownloadRecords/records/",
+			"targetTable": "filedownloadrecords"
+		},
+		{
+			"name": "aclsnapshots",
+			"description": "This job processes the acl snapshot data",
+			"scriptName": "acl_snapshots.py",
+			"sourcePath" : "log.sagebase.org/aclSnapshots/records/",
+			"targetTable": "aclsnapshots"
+		},
+		{
+			"name": "verificationsubmissionsnapshots",
+			"description": "This job processes the verification submission snapshot data",
+			"scriptName": "verification_submission_snapshots.py",
+			"sourcePath" : "log.sagebase.org/verificationSubmissionSnapshots/records/",
+			"targetTable": "verificationsubmissionsnapshots"
+		},
+		{
+			"name": "certifiedquizSnapshots",
+			"description": "This job processes the user's certified quiz snapshots",
+			"scriptName": "certified_quiz_snapshots.py",
+			"sourcePath" : "log.sagebase.org/certifiedUserPassingSnapshots/records/",
+			"targetTable": "certifiedquizsnapshots"
+		},
+		{
+			"name": "certifiedquizquestionsnapshots",
+			"description": "This job processes the user's certified quiz question snapshots",
+			"scriptName": "certified_quiz_question_snapshots.py",
+			"sourcePath" : "log.sagebase.org/certifiedUserPassingSnapshots/records/",
+			"targetTable": "certifiedquizquestionsnapshots"
+		},
+		{
+			"name": "accessrequirementsnapshots",
+			"description": "This job processes the access requirement snapshot data",
+			"scriptName": "access_requirement_snapshot.py",
+			"sourcePath" : "log.sagebase.org/accessRequirementSnapshots/records/",
+			"targetTable": "accessrequirementsnapshots"
+		},
+		{
```

Comment:
Does filesinventory belong here?

Reply:
Nope, since there is not ETL job for it (AWS generates this data already in parquet)